app/Config/core.php

```diff
@@ -12,28 +12,29 @@
  * Redistributions of files must retain the above copyright notice.
  *
  * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
  * @link          http://cakephp.org CakePHP(tm) Project
- * @package       app.Config
  * @since         CakePHP(tm) v 0.2.9
+ *
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 
-/**
- * CakePHP Debug Level:
- *
- * Production Mode:
- * 	0: No error messages, errors, or warnings shown. Flash messages redirect.
- *
- * Development Mode:
- * 	1: Errors and warnings shown, model caches refreshed, flash messages halted.
- * 	2: As in 1, but also with full debug messages and SQL output.
- *
- * In production mode, flash messages redirect after a time interval.
- * In development mode, you need to click the flash message to continue.
- */
-	Configure::write('debug', 2);
+    /**
+     * CakePHP Debug Level:.
+     *
+     * Production Mode:
+     * 	0: No error messages, errors, or warnings shown. Flash messages redirect.
+     *
+     * Development Mode:
+     * 	1: Errors and warnings shown, model caches refreshed, flash messages halted.
+     * 	2: As in 1, but also with full debug messages and SQL output.
```

Comment:
__[staging][code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed